src/main/webapp/register.html

```diff
@@ -66,12 +66,13 @@ <h5 class="text-center lead">Fill in the following fields below.</h5>
         <div class="form-row">
           <div class="col-md-6 mb-3">
             <label for="company-field">Company</label>
-            <input type="text" class="form-control" id="company-field" placeholder="Google" 
+            <input type="text" class="form-control" id="company-field" placeholder="Company"
                 value="" name="company">
           </div>
           <div class="form-group col-md-6 mb-3">
             <label for="job-field">Job</label>
             <select class="form-control" id="job-field" name="job">
+              <option></option>
```

Comment:
Why this blank option?

Reply:
This is so they can choose to not select a job, say if they are looking to apply to their first job and don't currently have a title.